warehouse_plugin: "warehouse_ros_mongo::MongoDatabaseConnection"
warehouse_host: "localhost"
warehouse_port: 33829
